src/app/item-page/simple/item-types/publication/publication.component.html

```diff
@@ -106,6 +106,15 @@
       [fields]="['datacite.relation.isReferencedBy']"
       [label]="'item.page.referenced'">
     </ds-item-page-uri-field>
+    <!-- Below is an example of how to render one or more lat/lng points and/or bounding box rectangles
+         in a tiled map viewer. Set 'cluster' to true for marker clustering -->
+<!--    <ds-geospatial-item-page-field [item]="object"-->
+<!--                                   [label]="'item.page.places'"-->
+<!--                                   [pointFields]="['dcterms.spatial']"-->
+<!--                                   [bboxFields]="['external.spatial.bbox']"-->
```

Comment:
Is this a typo?  In the component you have this `bboxFields` set to `gnd.coordinates.bbox` by default.  But, as noted above, it doesn't exist in DSpace

Reply:
i've changed this field default to dcterms.spatial